toolchain/diagnostics/diagnostic_consumer.cpp

```diff
@@ -19,8 +19,15 @@ auto StreamDiagnosticConsumer::HandleDiagnostic(Diagnostic diagnostic) -> void {
   for (const auto& message : diagnostic.messages) {
     message.loc.FormatLocation(*stream_);
     *stream_ << ": ";
-    if (message.level == DiagnosticLevel::Error) {
-      *stream_ << "ERROR: ";
+    switch (message.level) {
+      case DiagnosticLevel::Error:
+        *stream_ << "ERROR: ";
+        break;
+      case DiagnosticLevel::Warning:
+        *stream_ << "WARNING: ";
+        break;
+      case DiagnosticLevel::Note:
+        break;
```

Comment:
Could you split this out to its own PR? Mostly because it causess some churn in diagnostics and it seems a bit cleaner to have those separate from the import part.

Reply:
Don3, #3972